druid/examples/blocking_function.rs

```diff
@@ -0,0 +1,100 @@
+use druid::ExtEventSink;
```

Comment:
can we include the same copyright notice we do in other examples?

I would also like a `//! top level doc comment` that explains in a few words what this example is doing and why.